### PR TITLE
Remove completed field and add sent to provider at

### PR DESCRIPTION
**Pull request by plisovin**
### Trello card

[Trello-4999](https://trello.com/c/5nKyYNRw/4999-support-applys-continuous-applications-with-necessary-changes-to-the-git-api)

### Context

Removed the deprecated filed ```completed``` and add new filed ```sent_to_provider_at```

### CODE REVIEW

File: GetIntoTeachingApiTests/Contracts/Output/ApplyCandidateApi/sync_with_an_existing_candidate_and_fully_populated_application_form.json [hunk 1]
```diff
@@ -92,6 +92,10 @@
       },
       {
         "Key": "dfe_name",
+        "Value": "0001-01-01T00:00:00"
```
Comment (asatwal) [inline GetIntoTeachingApiTests/Contracts/Output/ApplyCandidateApi/sync_with_an_existing_candidate_and_fully_populated_application_form.json:95]: Same comment as above ie expected a `dfe_name`

**Review by asatwal — DISMISSED** [review]
👍 LGTM. Just a few comments on what seems to be test values and making them realistic, if I've understood correctly.

**Review by asatwal — APPROVED** [review]
👍

**Comment by martyn-w** [comment]
Closing as this has been open since 2023

**Pull request closed without merging**